sample_consensus/include/pcl/sample_consensus/impl/mlesac.hpp

```diff
@@ -116,10 +117,10 @@ pcl::MaximumLikelihoodSampleConsensus<PointT>::computeModel (int debug_verbosity
     std::vector<double> p_inlier_prob (indices_size);
     for (int j = 0; j < iterations_EM_; ++j)
     {
+      const double prob_factor2 = gamma / (sqrt (2 * M_PI) * sigma_);
```

Comment:
Would a name like weighted_normalization_factor make more sense?

It might make sense to separate out the denominator: the normalization_factor and compute it outside the loop.

Reply:
Renamed and also separated `normalization_factor`, although `prob_factor2` (now `weighted_normalization_factor`) is not computed that often (at most a few hundred times, I think).